sandboxed_containers/understanding-sandboxed-containers-workloads.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: ASSEMBLY
+[id="understanding-sandboxed-containers-workloads"]
+= Understanding {sandboxed-containers-first} workloads
+include::_attributes/common-attributes.adoc[]
+:context: understanding-sandboxed-containers-workloads
```

Comment:
Yeah here - you changed the context, so the condition statement no longer works.

Reply:
The condition in the FIPS module has the following: `ifeval::["{context}" == "understanding-sandboxed-containers"]`

It's referencing context that no longer exists, since it was changed, so it isn't pulling the OSC lines in.

Reply:
Thanks Miriam.